resources/authgear/templates/pl/translation.json

```diff
@@ -1252,7 +1252,7 @@
   "v2.page.settings-oob-otp-sms.default.title": "{PhoneOTPMode, select, whatsapp {WhatsApp} whatsapp_sms {WhatsApp/SMS} other {SMS} } wiadomość",
   "v2.page.settings-oob-otp.default.added-at-label": "Dodano o <span data-date=\"{rfc3339}\" data-date-type=absolute data-date-date-style=long>{time, datetime, short} UTC</span>",
   "v2.page.settings-oob-otp.default.oob-otp-email-button-label": "+ Dodaj email",
-  "v2.page.settings-oob-otp.default.oob-otp-sms-button-label": "+ Dodaj numer telefonu {PhoneOTPMode, select, whatsapp {WhatsApp} whatsapp_sms {WhatsApp/SMS} other {SMS} } Passkey",
```

Comment:
I guess this Passkey is incorrect so I removed it, although I don't know this language